os.bpmsuite.common/added/launch/bpmsuite-security-ldap.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+########## Environment Variables ##########
+
+function unset_kie_security_ldap_env() {
+    # please keep these in alphabetical order
+    unset KIE_AUTH_LDAP_ALLOW_EMPTY_PASSWORDS
+    unset KIE_AUTH_LDAP_BASE_CTX_DN
+    unset KIE_AUTH_LDAP_BASE_FILTER
+    unset KIE_AUTH_LDAP_BIND_CREDENTIAL
+    unset KIE_AUTH_LDAP_BIND_DN
+    unset KIE_AUTH_LDAP_DEFAULT_ROLE
+    unset KIE_AUTH_LDAP_DISTINGUISHED_NAME_ATTRIBUTE
+    unset KIE_AUTH_LDAP_JAAS_SECURITY_DOMAIN
+    unset KIE_AUTH_LDAP_PARSE_ROLE_NAME_FROM_DN
+    unset KIE_AUTH_LDAP_PARSE_USERNAME
+    unset KIE_AUTH_LDAP_REFERRAL_USER_ATTRIBUTE_ID_TO_CHECK
+    unset KIE_AUTH_LDAP_ROLE_ATTRIBUTE_ID
+    unset KIE_AUTH_LDAP_ROLE_ATTRIBUTE_IS_DN
+    unset KIE_AUTH_LDAP_ROLE_FILTER
+    unset KIE_AUTH_LDAP_ROLE_NAME_ATTRIBUTE_ID
+    unset KIE_AUTH_LDAP_ROLE_RECURSION
+    unset KIE_AUTH_LDAP_ROLES_CTX_DN
+    unset KIE_AUTH_LDAP_SEARCH_SCOPE
+    unset KIE_AUTH_LDAP_SEARCH_TIME_LIMIT
+    unset KIE_AUTH_LDAP_URL
+    unset KIE_AUTH_LDAP_USERNAME_BEGIN_STRING
+    unset KIE_AUTH_LDAP_USERNAME_END_STRING
+}
+
+function add_module() {
+    local xml=$1
+    local name=$2
+    local envVar=$3
+
+    if [[ ! -z ${envVar} ]]; then
+        echo ${xml} '<module-option name="'${name}'" value="'${envVar}'"/>'
+    else
+        echo ${xml}
+    fi
+}
+
+function configure_ldap_security_domain() {
+    if [[ -z ${KIE_AUTH_LDAP_URL} ]]; then
```

Comment:
If user provides only this env the configuration will be set, plus I didn't find default values for any env that does not need a explicit value from user.

Reply:
I don't think default values would help as the LoginModule already has their own and are documented.